website/static/js/institutionSignIn.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+var ko = require('knockout');
+
+var $osf = require('js/osfHelpers');
+
+var ViewModel = function() {
+    var self = this;
+    self.instNames = ko.observableArray([]);
+    self.selectedInst = ko.observable();
+    self.insts = {};
+    $osf.ajaxJSON(
+        'GET',
+        window.contextVars.apiV2Prefix + 'institutions/',
+        {
+            isCors: true
+        }
+    ).done( function(response){
+        for (var i = 0; i < response.data.length; i++){
```

Comment:
Ah! I think a bunch of my previous comments got lost when this code was moved to a new file. I had suggested refactoring this to use `response.data.map`.